### ISSUE TITLE
Bagnon has been blocked from an action only available to Blizzard UI

### ISSUE THREAD

**Issue by Ischnaroth** [Jaliborc/Bagnon#1901]
**Which software were you running?**
 - Addon version name: Bagnon 10.2.7 installed with CurseForge
 - Client used: WoW Retail 10.2.7
**Have you read the changelog?**
 Mark if read. Yes

**Please describe the bug.**
After a few specific processes (which will be described later), the action bars no longer work and the error message keeps popping up: Bagnon has been blocked. The requested action is reserved for the Blizzard UI.

**Please describe how to reproduce it.**
1. Create a new Character
2. Play the Game until Level 10 (that you can Spend Talent-Points)
3. Spend one Talent Point and Confirm
4. Ride a Dragon-Riding Mount or go in an Dungeon/Szenario
5. Try to use a Spell or Action with Hotkeys / Keyboard / Mousekeys
6. Error shows up

**Any Screenshots?**
No Screenshots for this Error

**Error Logs are Important!**
Unfortunately no logs are displayed to me

**Comment by Firsthunterz** [comment]
This happens to me on Remix as well. Every time I spend talent points when i lvl will trigger this as well as after dismounting dragon riding mount. It also happens when u zone into an instance. It stops when you hit lvl 70 though.

**Comment by Mewohkie** [comment]
Having this happen a lot in remix when I first get into a dungeon. After the first mobs, bagnon blocks me from pressing 1 through = on my keyboard to use my action bar, I can only use it if I click on the action bar icon with the mouse. Getting real sick of having to do a /reload in the beginning of a dungeon as the tank is b-lining for the last boss without slowing down.

**Comment by Riuujin** [comment]
same issue, level in dungeon > pick talent > exit dungeon > action bar with keyboard trigger no longer working

**Comment by Mewohkie** [comment]
Still happening after the latest update

**Comment by bugzfq** [comment]
Yeah. Still happening till version is 11.0.2. Only use `/rl` to resolve it temporarily but I do not want do this action frequently
